manifests/init.pp

```diff
@@ -61,6 +61,13 @@
 #   because Vault can block a scheduler thread". Default: number of CPUs
 #   on the system, retrieved from the ``processorcount`` Fact.
 #
+# * `api_addr`
```

Comment:
Could you please document this in the top level `README.me` as well?

Reply:
done